src/core/midiMsg.cpp

```diff
@@ -63,4 +65,15 @@ unsigned int MidiMsg::getMessageLength(){
 std::string MidiMsg::getMessageSender(){
 	return m_sender;
 }
+
+//-------------------------- PRIVATE MEMBER FUNCTIONS --------------------------
+
+void MidiMsg:fixVelocityZero(){
+	if (MMF_NOTEON.check(self)){
+		if (getByte(2) == 0){
+			m_message[0] |= 0b00010000;
```

Comment:
Yes this is terrible.. Shortly I'll introduce some functions for friendly MidiMsg manipulation.